integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/transport/test/JMSMapMessageTestCase.java

```diff
@@ -95,4 +108,32 @@ public void testJMSProxy() throws Exception {
     public void destroy() throws Exception {
         super.cleanup();
     }
+
+    private Callable<Boolean> isMessagesConsumed(final JMSQueueMessageConsumer consumer) {
+
+        return new Callable<Boolean>() {
+            @Override
+            public Boolean call() throws Exception {
+
+                Message msg = consumer.popRawMessage();
+                if (msg != null) {
```

Comment:
Do we need this null check since we are doing an instanceof after that?

Reply:
I will remove the null check since  instanceof checks the null check as well.